src/mon/AuthMonitor.cc

```diff
@@ -1870,6 +1891,80 @@ bool AuthMonitor::prepare_command(MonOpRequestRef op)
   return false;
 }
 
+/* Generate the caps that should be present in the entity's auth keyring
+ * after running the "fs authorize" command. This is done by merging the
+ * caps already present in the client's auth keyring with the new caps
+ * provided by the user at "fs authorize" command.
+ */
+int AuthMonitor::_gen_wanted_caps(EntityAuth& e_auth,
```

Comment:
return should be enum type.

Reply:
Done. 

I'll say the same thing here too -- this much more complicated than simply using `int`; especially because this `enum` is not used beyond the very limited context of _gen_wanted_caps().